packages/palette_generator/lib/palette_generator.dart

```diff
@@ -620,7 +620,8 @@ class PaletteTarget with Diagnosticable {
 
   @override
   bool operator ==(dynamic other) {
```

Comment:
You can change this to `bool operator==(Object other)` safely, since that is the signature of `Object.==`